Dnn.AdminExperience/Dnn.PersonaBar.Extensions/Components/ConfigConsole/ConfigConsoleController.cs

```diff
@@ -52,7 +52,7 @@ public string GetConfigFile(string configFile)
             }
             else
             {
-                var doc = File.ReadAllText(Path.Combine(Globals.ApplicationMapPath, "\\", configFile));
+                var doc = File.ReadAllText(Path.Combine(Globals.ApplicationMapPath, configFile));
```

Comment:
```suggestion
                var doc = File.ReadAllText(Path.Combine(Globals.ApplicationMapPath, "\\", configFile));
```

Reply:
This is already done in my previous commit.  :P  
https://github.com/dnnsoftware/Dnn.Platform/pull/4172/commits/7fa48b6d811ad51b22aadf0e1f6122d9636610b2
Also, breathe.  :)

Reply:
![image](https://user-images.githubusercontent.com/938023/95690894-d3886800-0bcf-11eb-8a49-362e36128f43.png)

Reply:
@david-poindexter Here's a video showing it.  Please let me know if I'm missing something and I'll make it happen.  https://www.youtube.com/watch?v=33-1_VW-2PA

Reply:
@hismightiness please see my suggested change.  The `\\` is needed in line 55.
```
var doc = File.ReadAllText(string.Concat(Globals.ApplicationMapPath, "\\", configFile));
```

I have been actually testing your code, so I am certain of it.

Reply:
I don't know what's being missed here... My last commit in this PR already did that. 
![image](https://user-images.githubusercontent.com/938023/95691129-ad63c780-0bd1-11eb-9d2d-c8203f9ce113.png)

Reply:
Here's me debugging it, if it helps. 🤔 
https://www.youtube.com/watch?v=csL2cMHfqhs

Reply:
Thanks for the video - I see what was wrong now on my end.  Too many branches and trying to sync your changes with my totally different way of "fixing this".  I'll approve and merge now.